docs/source/docs/command-reference/context_free/transform.md

```diff
@@ -101,7 +101,9 @@ Basic dataset item manipulations:
 - [`remove_images`](#remove_images) - Removes specific images
 - [`remove_annotations`](#remove_annotations) - Removes annotations
 - [`remove_attributes`](#remove_attributes) - Removes attributes
-- [`astype_annotations`](#astype_annotations) - Convert annotation type
+- [`correct`](#correct) - Corrects annotaiton types
+- [`astype_annotations`](#astype_annotations) - Transforms annotation types
+- [`clean`](#clean) - Remove noisy data for tabular dataset
```

Comment:
Remove -> Removes

Reply:
Fix in [c79dae8](https://github.com/openvinotoolkit/datumaro/pull/1599/commits/c79dae8f38c555150d38887fcd08379ed5a0b5c2)